backend/src/main/java/ai/verta/modeldb/configuration/Migration.java

```diff
@@ -13,10 +14,7 @@ public class Migration {
   private static final Logger LOGGER = LogManager.getLogger(Migration.class);
 
   public static boolean migrate(MDBConfig mdbConfig) throws Exception {
-    var liquibaseMigration =
-        Boolean.parseBoolean(
-            Optional.ofNullable(System.getenv(CommonConstants.ENABLE_LIQUIBASE_MIGRATION_ENV_VAR))
-                .orElse("false"));
+    var liquibaseMigration = new MigrationSetupConfig().isMigration();
```

Comment:
rather than constructing this here, can you pass in a `MigrationSetupConfig` instance that wraps the MDBConfig and provides everything that is needed for doing migrations?

Reply:
Improve code with better Bean creation